build/build.xml

```diff
@@ -69,11 +69,8 @@
 	<target name="compile" depends="init" description="compile the source">
 		<echo message="Compiling the source..." />
 		
-		<!-- Compile with debug information if the property "javac.debug" is set to true -->
 		<local name="debug" />
-		<condition property="debug" value="true" else="false">
-			<istrue value="${javac.debug}" />
-		</condition>
+		<property name="debug" value="true" />
```

Comment:
The property could be dropped (and set the value directly in the task attribute)?